ignf_gpf_api/_conf/default.ini

```
@@ -133,6 +133,23 @@ check_execution_logs=${routing:check_execution_get}/logs
 
 # fichiers static
 static_list=${store_api:root_datastore}/statics
+static_get=${routing:static_list}/{static}
+static_upload=${routing:static_list}/{static}
+static_delete=${routing:static_list}/{static}
+static_re_upload=${routing:static_list}/{static}
+static_partial_edit=${routing:static_list}/{static}
+static_download=${routing:static_get}/file
+
+# annexes
+annexes_list=${store_api:root_datastore}/annexes
+annexes_get=${routing:annexes_list}/{annexes}
+annexes_upload=${routing:annexes_get}
+annexes_delete=${routing:annexes_get}
+annexes_re_upload=${routing:annexes_get}
+annexes_partial_edit=${routing:annexes_get}
+annexes_download=${routing:annexes_get}/file
+annexes_publish_by_label=${routing:annexes_list}/publication
+annexes_unpublish_by_label=${routing:annexes_list}/unpublication
```

Pour que se soit homogène avec le reste, à mettre au singulier (NB : il faudra modifier les tests du coup).
```suggestion
annexe_list=${store_api:root_datastore}/annexes
annexe_get=${routing:annexe_list}/{annexes}
annexe_upload=${routing:annexe_get}
annexe_delete=${routing:annexe_get}
annexe_re_upload=${routing:annexe_get}
annexe_partial_edit=${routing:annexe_get}
annexe_download=${routing:annexe_get}/file
annexe_publish_by_label=${routing:annexe_list}/publication
annexe_unpublish_by_label=${routing:annexe_list}/unpublication
```